pkgs/by-name/lo/lorem/package.nix

```
@@ -17,20 +17,20 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "lorem";
-  version = "1.4";
+  version = "1.5";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
     owner = "World/design";
     repo = "lorem";
     rev = finalAttrs.version;
-    hash = "sha256-6+kDKKK1bkIOZlqzKWpzpjAS5o7bkbVFITMZVmJijuU=";
+    hash = "sha256-q6gpxxNebf2G/lT5wWXT/lVp3zR8QLWB8/sdK+wLTJ8=";
   };
 
   cargoDeps = rustPlatform.fetchCargoTarball {
```

I don't think this hash is right for this fetcher. Let's switch to the stable `fetchCargoVendor` fetcher like on `master`

```suggestion
  cargoDeps = rustPlatform.fetchCargoVendor {
```